code-challenges.js

```diff
@@ -1,63 +1,117 @@
-// ASSESSMENT 2: Coding Practical Questions
-
-// --------------------1) Create a function that takes a number as an argument and decides if the number is evenly divisble by three or not.
-// Use the test variables provided.
+// // ASSESSMENT 2: Coding Practical Questions
+//
+// // --------------------1) Create a function that takes a number as an argument and decides if the number is evenly divisble by three or not.
+// // Use the test variables provided.
+//
+//
 
 var num1 = 15
-// Expected output: "15 is divisible by three"
-
 var num2 = 0
-// Expected output: "0 is divisible by three"
-
 var num3 = -7
-// Expected output: "-7 is not divisble by three"
-
-
-
-
-
-// --------------------2) Create a function that takes in the randomNouns variable and returns an array with all the words capitalized.
-
+//
+// // put a const so I can console log it later
+const divisibleOrnot = (num) => {
+// // Create an equation so whenever I have a no remainder it'll spit out is divisible by 3
+    if(num % 3 == 0){
+        return `${num} Divisible by 3`
+    }else{
+        return `${num} Not divisible by 3`
+    }
+}
+// // these syntaxes are gonna be the end of me
+console.log(divisibleOrnot(num1));
+console.log(divisibleOrnot(num2));
+console.log(divisibleOrnot(num3));
+//
+//
+// // Expected output: "-7 is not divisble by three"
+// // Expected output: "15 is divisible by three"
+// // Expected output: "0 is divisible by three"
+//
+//
+//
+// // --------------------2) Create a function that takes in the randomNouns variable and returns an array with all the words capitalized.
+//
 var randomNouns = ["streetlamp", "potato", "teeth", "conclusion", "nephew", "temperature", "database"]
-// Expected output: ["Streetlamp", "Potato", "Teeth", "Conclusion", "Nephew", "Temperature", "Database"]
-
-
-
-
+//
+// // I'm declaring a function that takes in this array ^ and returns the first letter capitalized.
+  const capitalizer = (array) => {
+// // So I have to declare a function that takes in the array
+    let eachItem = array.map(value => {
+// // Then access the first index and capalize the letter
+    return value[0].toUpperCase() + value.substring(1)
+  })
+// // return the local variable with a .join so itll go back into a string
+  return eachItem.join(" ")
```

Comment:
So the expected output, requires this to be an array, there is no need to .join it. Be sure to double check the output requirements!

> Create a function that takes in the randomNouns variable and *returns an array* with all the words capitalized.